src/odemis/dataio/png.py

```diff
@@ -41,7 +41,7 @@ def _saveAsPNG(filename, data):
     if data.metadata.get(model.MD_DIMS) == 'YXC':
         rgb8 = data
     else:
-        data = img.ensure2DImage(data)
+        data = img.ensure2DImage(data)  # Note: this is called if we do not have MD_DIMS but already have an RGB image --> raises ValueError though maybe not wanted!! Should we add an additional check for RGB?
```

Comment:
In general, the rule is that if a DataArray is RGB, it *must* have MD_DIMS=YXC. I could imagine that is some special cases, that's inconvenient or not useful, but here I'd suggest to just stick to that rule. If you want to export RGB into a PNG, just make sure the metadata is correct, and the exporter will do the right thing.